.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+# CircleCI is used only for Ansible-Lint and Ansible Syntax checks
+version: 2
+jobs:
+  build:
+    docker:
+      - image: ansible/ansible:ubuntu1604
```

Comment:
I recommend using the image `williamyeh/ans:ubuntu16.04` instead.

Surprisingly, `ansible/ansible:ubuntu1604` does not provide `ansible-playbook` or anything else ansible related.. well, except for `/etc/ansible/hosts` (woo!):

```
root@d8d812457280:/opt# find / -name '*ansible*'
/etc/ansible
```